### PR TITLE
Simplify tracker integration

### LINKED ISSUE #2: Missing doc on howto link tracker with consent
Hi,

Thank you for your sharing. However, I could not find neither in readme nor in option file how to link (sync) the consent to launching the cookies/trackers.

### PR DESCRIPTION
**Pull request by BuonOmo**
1. Give access to interface at the class level, not needing instance everywhere.
2. Create a `change` event, needed when specific `acceptedCategories` are set.
3. Pass CookieConsent instance to events for convenience.
4. Rewrite example with a google analytics intregration.
5. Add minified code for usage with cdns such as jsdelivr.

Fixes #2

### CODE REVIEW

**Review by Quiwin — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ It is designed to help you comply with data privacy laws like [EU Cookie Law][eu
...
+or
+
+```bash
+yarn add @klaxit/cookie-consent
+```
+
+or
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@klaxit/cookie-consent@0.3.1/dist/cookie-consent.js"></script>
```
Comment (ccyrille) [inline README.md]: Pour ne pas avoir à mettre à jour en permanence (et oublier de le faire) j'ai l'impression que ça serait mieux d'indiquer : 

```suggestion
<script src="https://cdn.jsdelivr.net/npm/@klaxit/cookie-consent@X.Y.Z/dist/cookie-consent.js"></script>
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,26 @@ It is designed to help you comply with data privacy laws like [EU Cookie Law][eu
...
+```
+
+<sub>**NOTE:** the latter isn't advised if you want to have one build for all your libraries. Mostly great for tests</sub>
```
Comment (ccyrille) [inline README.md]:
```suggestion
<sub>**NOTE:** the latter isn't advised if you want to have one build for all your libraries. Mostly great for tests. And don't forget to replace X.Y.Z with current package version!</sub>
```

**Review by ccyrille — APPROVED** [review]
Just one detail on the README but all good for me on the code 👍 👏

**Pull request merged**